package.json

```diff
@@ -90,7 +90,6 @@
     "debug": "2.2.0",
     "minimist": "1.2.0",
     "ramda": "0.21.0",
-    "rollup": "0.37.0",
```

Comment:
I concur. However not sure of the sanest (least path to not pissing people off) way to do this is. Any suggestions are welcome @YuryStrozhevsky